api/resolvers/notifications.js

```diff
@@ -346,6 +363,11 @@ export default {
           "Invoice"."actionType" = 'POLL_VOTE' OR
           "Invoice"."actionType" = 'BOOST'
         )
+        AND EXISTS (
+          SELECT 1
+          FROM "ItemAct"
+          WHERE "ItemAct"."invoiceId" = "Invoice".id
+        )
```

Comment:
when we have a failed invoiced action that it is then retried successfully with a fee credit or reward sats payment, we have this situation where there is a failed invoice that used to be bound to this action but it is not anymore. 
This check is to exclude it from notifications. Maybe there are other ways, but i am unsure.